src/components/Accodian/index.js

```diff
@@ -9,30 +9,34 @@ function BasicExample() {
           Where is Yatra Nepal Located?
         </Accordion.Header>
         <Accordion.Body>
-          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
-          eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad
-          minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+        Yatra Nepal is Located on Basbari Road (In Front of British Gurkha College), Kathmandu.
         </Accordion.Body>
       </Accordion.Item>
       <Accordion.Item eventKey="1">
         <Accordion.Header>What are services avilable?</Accordion.Header>
         <Accordion.Body>
-          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
-          eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad
+        Yatra Nepal Provides:<br/>
+Periodic Service<br/>
```

Comment:
Make sure we are using same typography, font-family as mentioned in design.
These text are not consistent with font-family